src/components/BlipGroupCard.vue

```diff
@@ -27,7 +27,8 @@
         />
 
         <div class="flex" :class="'group-notification ' + group.position" v-if="group.date && group.hasNotification">
-          <img v-if="group.status === 'accepted' && group.position === 'right'" :src="checkSentSvg"/>
+          <img v-if="group.status === 'waiting' && group.position === 'right'" :src="clockSvg">
```

Comment:
Since the event `waiting` doesn't exists on Lime, this can lead to confusion due to inconsistency. It would be better if the clock appear if the value of `status` is `undefined`

Reply:
i thought on that too, but this way there would be no way to render a card without a status

Reply:
I changed it to use the dispatched status and to show it even if there is no date stored yet